Index: Map Creation Tool.csproj.user
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"utf-8\"?>\r\n<Project ToolsVersion=\"Current\" xmlns=\"http://schemas.microsoft.com/developer/msbuild/2003\">\r\n  <ItemGroup>\r\n    <Compile Update=\"src\\View\\Form1.cs\">\r\n      <SubType>Form</SubType>\r\n    </Compile>\r\n  </ItemGroup>\r\n</Project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Map Creation Tool.csproj.user b/Map Creation Tool.csproj.user
--- a/Map Creation Tool.csproj.user	(revision 4eeb3e7272463f73ba4b7f99fffe09b19387d54e)
+++ b/Map Creation Tool.csproj.user	(date 1742839632816)
@@ -4,5 +4,8 @@
     <Compile Update="src\View\Form1.cs">
       <SubType>Form</SubType>
     </Compile>
+    <Compile Update="src\View\MapCreationForm.cs">
+      <SubType>Form</SubType>
+    </Compile>
   </ItemGroup>
 </Project>
\ No newline at end of file
